ext/mysqli/mysqli.c

```diff
@@ -428,19 +431,26 @@ HashTable *mysqli_object_get_debug_info(zval *object, int *is_temp)
 	mysqli_object *obj = Z_MYSQLI_P(object);
 	HashTable *retval, *props = obj->prop_handler;
 	mysqli_prop_handler *entry;
+	int er;
 
 	retval = zend_new_array(zend_hash_num_elements(props) + 1);
 
+	er = EG(error_reporting);
+	EG(error_reporting) = 0;
```

Comment:
Would it be feasible to instead pass in an extra `zend_bool quiet` parameter to suppress the generation of warnings (or exceptions in the future)?

Reply:
Generally for `BP_VAR_IS`, not just `get_debug_info()`. Doing `isset($mysqli->whatever)` also shouldn't warn/throw.

Reply:
I think I managed to do it! :) Now, I'm confirming it with the CI...

Reply:
Now, it works, so I am currently trying to address the last remaining failing test: https://travis-ci.org/php/php-src/jobs/642786841#L1844